### ISSUE TITLE
The final word in a list item is not announced correctly by VoiceOver

### ISSUE THREAD

**Issue by wilsond-gds** [alphagov/reported-bugs#73]
**Upstream bug:** [https://bugs.webkit.org/show_bug.cgi?id=246155](https://bugs.webkit.org/show_bug.cgi?id=246155)
**Date:** 2022-10-06
**Reported by:** Derren Wilson
**Related to:** Found while testing Digital Identity app integration

---

### Overview

Load a HTML page with list items and use the VO (control+option) + A shortcut to instruct Voice Over to read out the content with Safari version 16.0 (17614.1.25.9.10, 17614)

### Reduced test case

See video below.

### Expected result

Each word in the list item is read out in full.

### Actual result

The final word in each list item is split into two - what seems to be the final three characters is separated from the rest of the word. This seems to happen before the text is fed to the voice synthesiser, so a word like 'first' would be read out as 'fir' 'saint'.

https://user-images.githubusercontent.com/101639632/194339342-7505bbd9-280d-4dcd-a7f3-5e73bbce5252.mov

**Comment by wilsond-gds** [comment]
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title>Document</title>
    </head>
    <body>

    <p>Paragraph within body element</p>

        <ol>
<!-- will be read out as 'Unordered list i... tem' -->
            <li>Unordered list item</li>
            <li>Unordered list item</li>
            <li>Unordered list item</li>
        </ol>

        <ul>
            <li>Ordered list item</li>
            <li>Ordered list item</li>
            <li>Ordered list item</li>
        </ul>
        <details>
            <summary>Details heading</summary>
            <p>Details content</p>
            <ol>
                <li>The word 'by' will be read out correctly: by</li>
                <li>The word 'app' will be read out incorrectly: app&nbsp;</li>
                <li>The word 'app' will be read out incorrectly: app&nbsp;&nbsp;</li>
                <li>The word 'app' will be read out correctly but with an extra word 'space' announced at the end: app&nbsp;&nbsp;&nbsp;</li>
                <li>Unordered list item within details</li>
                <li>Unordered list item within details</li>
                <li>Unordered list item within details</li>
            </ol>
        </details>
    </body>
</html>
```

**Comment by querkmachine** [comment]
This issue was apparently solved by a different bugfix the WebKit team made in October 2022. I have been unable to reproduce it in Safari 26.4, so I'm going to close this as having been fixed.
